tests/test_encvector.py

```diff
@@ -0,0 +1,16 @@
+import pysdsl
+import pytest
+
+
+@pytest.mark.parametrize("Type", pysdsl.all_compressed_integer_vectors)
+def test_encvector(Type):
+    if Type == pysdsl.pysdsl.DirectAccessibleCodesVectorDP or \
```

Comment:
is it to just pass the tests for now? It is better then to split the test and mark the broken one with pytest.mark.skip

Reply:
Also please link here corresponding issue.

Reply:
Here's the corresponding issue #16